src/components/CarouselCard/__tests__/carouselCard.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import CarouselCard from '../index';
+import CarouselImage from '../../CarouselImage';
+
+describe('<CarouselCard />', () => {
```

Comment:
add test for:
* should set the right state when an indicator is clicked
and try to test what happen in startAnimation method